### ISSUE TITLE
SLA time on book calculation looking incorrect for LPs on existing markets

### ISSUE THREAD

**Issue by davidsiska-vega** [vegaprotocol/vega#9951]
All LPs reporting time on book as `0`: 
![image](https://github.com/vegaprotocol/vega/assets/62546419/0b419a2b-8a2e-4f55-bfbb-b4bdd54e58a7)

@wwestgarth says that in the snapshot the epoch start time is reported as
```
"epochStartTime": "-6795364578871345152"
```
which "isn't great". 

Also each LP's "start of  committment time" is also "-6795364578871345152":
```
type slaPerformance struct {
	s                 time.Duration
	start             time.Time // <---- THIS FELLA HERE
	previousPenalties *sliceRing[*num.Decimal]

	lastEpochBondPenalty      string
	lastEpochFeePenalty       string
	lastEpochTimeBookFraction string
	requiredLiquidity         string
	notionalVolumeBuys        string
	notionalVolumeSells       string
}
```

**Comment by davidsiska-vega** [comment]
Or just let things be and then propose governance transfers out of the insurance pools.

**Comment by jeremyletang** [comment]
fixed in tag v0.73.2
